### PR TITLE
3438 remove 24px

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/3438

### CODE REVIEW

File: core/modules/dblog/css/dblog.css [hunk 1]
```diff
@@ -68,18 +68,22 @@
   width: 30px;
   height: 30px;
   border-radius: 15px;
+  background: #ccc url(../../../misc/message-32-info.png) no-repeat center center;
+  background-size: 16px;
```
Comment (klonos) [inline core/modules/dblog/css/dblog.css]: I think that the changes in this file are overlapping with https://github.com/backdrop/backdrop/pull/2426/files#diff-617c1d5b7fa1ac6b8db6e8b4ad9db7e9 (for https://github.com/backdrop/backdrop-issues/issues/3437)

**Comment by jenlampton** [comment]
They should be changing different files?

**Comment by klonos** [comment]
All I'm saying is that both PRs have the exact same diff for `core/modules/dblog/css/dblog.css`. I think that merging them will result in conflicts, but I could be wrong, ...in which case please ignore me 😅

**Comment by jenlampton** [comment]
Oh, you're saying this PR has the commit from the other issue on it... can fix!

**Comment by jenlampton** [comment]
Fixed :)

File: core/modules/system/css/messages.theme.css [hunk 2]
```diff
@@ -100,33 +106,11 @@ div.messages.error:before {
...
-  }
-  div.messages.error:before {
-    background-image: url(../../../misc/message-32-error.png);
-  }
-  div.messages.info:before {
-   background-image: url(../../../misc/message-32-info.png);
-  }
-}
-/* SVG overrides */
+/* SVG overrides
```
Comment (quicksketch) [inline core/modules/system/css/messages.theme.css]: Looks like you disabled the SVGs entirely here.

File: core/modules/system/css/messages.theme.css [hunk 3]
```diff
@@ -23,7 +23,7 @@
   height: 100%;
   background-position: center .625em;
   background-repeat: no-repeat;
-  background-size: 1.5em;
+  background-size: 32px;
```
Comment (quicksketch) [inline core/modules/system/css/messages.theme.css:26]: I think we should keep to `em` sizing for consistency within Basis, which uses em for everything I believe.

**Review by quicksketch — CHANGES_REQUESTED** [review]

File: core/modules/system/css/messages.theme.css (hunk 3 above)
Comment (quicksketch) [reply core/modules/system/css/messages.theme.css:26]: Nevermind, this is in `messages.theme.css`, not Basis. px is probably fine.

File: core/modules/system/css/messages.theme.css (hunk 2 above)
Comment (jenlampton) [reply core/modules/system/css/messages.theme.css]: oops, that was accidental. for testing. Fixing now!

Comment (jenlampton) [reply core/modules/system/css/messages.theme.css]: Fixed :)

File: core/modules/system/css/messages.theme.css (hunk 3 above)
Comment (jenlampton) [reply core/modules/system/css/messages.theme.css:26]: neverminding.

**Comment by jenlampton** [comment]
Pushed a new PR.

**Comment by quicksketch** [comment]
New sandbox.

**Review by quicksketch — APPROVED** [review]

**Pull request merged**